imap_processing/mag/l0/decom_mag.py

```diff
@@ -13,9 +13,9 @@
 
 from imap_processing import imap_module_directory
```

Comment:
I have a PR open to do all these changes for MAG 😢  I'm sorry I didn't merge it yet, I was prioritizing the SQS changes because I thought it would have more blockers. Since the changes in this PR would break the code, could you wait until I merge the MAG PR before merging this? I can finish it off today.

Reply:
oh darn..ok. I will wait for that.

Reply:
Sorry for the confusion!

Reply:
No worries. I totally forgot that that PR is open.